frontend/src/container/BillingContainer/utils.ts

```diff
@@ -0,0 +1,12 @@
+export const getRemainingDays = (billingEndDate: number): number => {
```

Comment:
it can be made generic and moved to timeUtils

Reply:
Done